### ISSUE TITLE
[Bug]: Unavailable videos are hidden in playlists

### ISSUE THREAD

**Issue by hix1000** [FreeTubeApp/FreeTube#5552]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

I noticed that there were some missing videos in a playlist I was looking at, (it showed it had 49 videos, but there were only 44)

### Expected Behavior

Unavailable videos on YouTube Show up on freetube

### Issue Labels

accessibility issue, API issue, content not loading, usability issue

### FreeTube Version

v0.21.3 Beta

### Operating System Version

Windows 11

### Installation Method

.exe

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
That's intentional, as YouTube hides them by default for logged out users too. The only difference in that YouTube numbers the videos as if the missing ones were still there, so there are some missing numbers, whereas FreeTube numbers based on what is actually visible.

**Comment by hix1000** [comment]
When I made this bug, the number didn't correlate with what was available.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
duplicate of https://github.com/FreeTubeApp/FreeTube/issues/3660
